dm/syncer/syncer.go

```diff
@@ -1387,9 +1389,37 @@ func (s *Syncer) syncDDL(queueBucket string, db *dbconn.DBConn, ddlJobChan chan
 
 		if !ignore {
 			var affected int
+			row, err2 := db.QuerySQL(s.syncCtx, s.metricsProxies, "SELECT @@TIMESTAMP")
```

Comment:
there are functions like `select UNIX_TIMESTAMP();` which returns a int32/int64 value.

And `GetSpecifiedColumnValuesAndClose` is used to get specific columns from result with unknown number of columns. In this function we can use `Scan` to avoid strconv.

And considering the convention of getDDLStatusFromTiDB

```
// createTime should be based on the timezone of downstream, and its unit is second.
```

You can also be careful about the timezone.